src/yoroi-wallets/cardano/ShelleyWallet.ts

```diff
@@ -131,6 +131,7 @@ export class ShelleyWallet implements WalletInterface {
   version: string
   checksum: CardanoTypes.WalletChecksum
   private _utxos: RawUtxo[]
+  stakingKeyPath: number[]
```

Comment:
private?
if private, remove from type

Reply:
also if it's known at initialization and never updated, please mark it as `private readonly stakingKeyPath: Readonly<Array<number>>>`

Reply:
@stackchain I have tested `private readonly stakingKeyPath: Readonly<Array<number>>` but it seems we can only add one `readonly` as `eslint` suggest.

I have added the initial `readonly`. Let me know if you think otherwise. Thanks!